src/main/modules/ardoq/RequestProcessor.ts

```diff
@@ -19,31 +24,66 @@ export class RequestProcessor {
   }
 
   public async processRequest(request: ArdoqRequest): Promise<Map<ArdoqComponentCreatedStatus, number>> {
```

Comment:
This method is pretty big now, could probably split some of it out

Reply:
I'll pick this up when I do the deletion feature as it's going to involve lots of work here anyway